src/main/host/process.c

```diff
@@ -318,7 +318,7 @@ static void _process_handleTimerResult(Process* proc, gdouble elapsedTimeSec) {
 #endif
 
 static void _process_getAndLogReturnCode(Process* proc) {
-    if(!proc->didLogReturnCode) {
+    if(!proc->didLogReturnCode && process_hasStarted(proc)) {
```

Comment:
Let's log a warning if the process hasn't started... maybe even an `error` so that it stand out more amongst the warnings about unimplemented syscalls. Unlike those, this happening in a "real" simulation is almost certainly a mistake.

Reply:
Added an error message to the beginning of `_process_getAndLogReturnCode()`.